.devcontainer/Dockerfile

```diff
@@ -310,14 +312,45 @@ RUN git clone https://github.com/OPENAIRINTERFACE/opencord.org.freeDiameter.git
     make -j"$(nproc)" && \
     make install && \
     cd / && \
-    rm -rf freediameter
+    rm -rf freediameter \
+    rm -rf /tmp/*
 
 ##### Go language server support for vscode
 RUN go get -v golang.org/x/tools/gopls
 
 #### Update shared library configuration
 RUN ldconfig -v
 
+
+##### Install Python requirements
+RUN pip3 install --quiet --upgrade --no-cache-dir "setuptools==49.6.0"
+
+### install python3-aioeventlet from the magma apt repo (from .circleci/config.yml)
+COPY orc8r/tools/ansible/roles/pkgrepo/files/jfrog.pub /tmp/
+RUN apt-key add /tmp/jfrog.pub \
+    && echo "deb https://facebookconnectivity.jfrog.io/artifactory/list/dev-focal/ focal main" > /etc/apt/sources.list.d/fbc.list \
+    && apt-get update -y  \
+    && apt-get install -y python3-aioeventlet \
+    && rm -f /tmp/* \
+    && rm -rf /var/lib/apt/lists/*  # delete lists that are downloaded by apt-get update
```

Comment:
Inconsistency regarding style: `&&` should be either pre- or post-pended everywhere. I prefer the prepended form, due to better visibility of the operation and less chance to miss it (see above).
However, check with the project guidelines please.

Reply:
I added a second commit to make style more consistent.